change/@ni-nimble-components-bcea55bf-ed59-4b11-b80b-6311749da460.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
I'm seeing a visual glitch when the drop down position is set to "above". The items that are scrolling behind the filter input are overlapping the line next to the input.

![Screenshot 2024-01-30 at 7 59 37 AM](https://github.com/ni/nimble/assets/10500124/1f59fb25-831a-412b-8f19-30577f4477c6)

Reply:
Definitely much better. It still looks a bit odd when zoomed way in because the top and bottom of the border behave differently. The top is either further back in the z-order or more transparent, so the items are still visible through/over it, while the bottom causes items to clip. But it's not really noticeable unless zoomed in so I'd say it's acceptable if it's hard to fix.


https://github.com/ni/nimble/assets/10500124/4ee21cec-68a4-46e9-aeb9-006a80e3b978

Reply:
Made one more style change that I think addresses this.